xinference/core/worker.py

```diff
@@ -31,16 +34,26 @@
 
 
 class WorkerActor(xo.Actor):
-    def __init__(self, supervisor_address: str, subpool_addresses: List[str]):
+    def __init__(
+        self,
+        supervisor_address: str,
+        main_pool: MainActorPoolType,
+        cuda_devices: List[int],
+    ):
         super().__init__()
         self._supervisor_address = supervisor_address
         self._supervisor_ref = None
         self._model_uid_to_model: Dict[str, xo.ActorRefType["ModelActor"]] = {}
         self._model_uid_to_model_spec: Dict[str, ModelDescription] = {}
-        self._subpool_address_to_model_uids: Dict[str, Set[str]] = dict(
-            [(subpool_address, set()) for subpool_address in subpool_addresses]
+
+        self._gpu_to_model_uids: Dict[int, Set[str]] = {
```

Comment:
I mean we should avoid running multiple models on a single device since it is a complex problem and many of our LLM runtimes assumes they are running on a exclusive device.

Reply:
> I mean we should avoid running multiple models on a single device since it is a complex problem and many of our LLM runtimes assumes they are running on a exclusive device.

The latest code has been changed to:  one card for one model